packages/storybook/cxl-lumo-styles/vaadin-tabs/vaadin-tabs[theme=cxl-tabs-slider].stories.js

```diff
@@ -0,0 +1,59 @@
+import { html } from 'lit-html';
+import '../../../cxl-ui/src/components/cxl-card';
+import archiveData from '../../cxl-ui/cxl-vaadin-accordion/theme=cxl-archive.data.json';
+
+export default {
+  title: 'CXL Lumo Styles/Elements/vaadin-tabs',
+};
+
+export const Default = () => html`
+  <style>
+    vaadin-tab {
+      width: calc(var(--cxl-wrap-width) / 2.5);
+    }
+  </style>
+
+  <vaadin-tabs orientation="horizontal" theme="cxl-tabs-slider minimal">
+    ${archiveData.map(
+      (el) =>
+        html`
+          <vaadin-tab>
+            <cxl-card theme="cxl-testimonial">
+              <article
+                class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+                id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+              >
+                <header class="entry-header">
+                  <img
+                    class="thumbnail"
+                    src="https://cxl.com/institute/wp-content/uploads/2020/05/48192546_10156982340630746_8127333122065825792_n-wpv_400pxx400px_center_center.jpg"
+                  />
+                  <img
+                    class="company-logo"
+                    src="https://cxl.com/institute/wp-content/uploads/2020/05/Screenshot-2020-05-27-at-13.28.50.png"
+                  />
+                  <h4>Kurt S.</h4>
+                  <p>Web Conversion Manager @ Pipedrive</p>
+                  <p>Estonia</p>
```

Comment:
These are nested into a `div.entry-byline` container, like in https://cxl.com/blog/marketing-without-email/